politeiawww/cmd/piwww/editproposal.go

```diff
@@ -23,14 +23,28 @@ import (
 // EditProposalCmd edits an existing proposal.
 type EditProposalCmd struct {
 	Args struct {
-		Token       string   `positional-arg-name:"token" required:"true"` // Censorship token
-		Markdown    string   `positional-arg-name:"markdownfile"`          // Proposal MD file
-		Attachments []string `positional-arg-name:"attachmentfiles"`       // Proposal attachments
+		Token       string   `positional-arg-name:"token" required:"true"`
+		Markdown    string   `positional-arg-name:"markdownfile"`
+		Attachments []string `positional-arg-name:"attachmentfiles"`
 	} `positional-args:"true" optional:"true"`
-	Random bool   `long:"random" optional:"true"` // Generate random proposal data
 	Name   string `long:"name" optional:"true"`
-	RFP    bool   `long:"rfp" optional:"true"`    // Insert a LinkBy timestamp to indicate an RFP
-	LinkTo string `long:"linkto" optional:"true"` // Censorship token of prop to link to
+	LinkTo string `long:"linkto" optional:"true"`
+	LinkBy int64  `long:"linkby" optional:"true"`
+
+	// Random can be used in place of editing proposal name & data. When
+	// specified, random proposal name & data will be created and submitted.
+	Random bool `long:"random" optional:"true"`
+
+	// RFP is a flag that is intended to make editing an RFP easier
+	// by calculating and inserting a linkby timestamp automatically
+	// instead of having to pass in a specific timestamp using the
+	// --linkby flag.
+	RFP bool `long:"rfp" optional:"true"`
+
+	// Usemd is a flag that is intended to make editing propsoal metadata easier
+	// by using exisiting proposal metadata values instead of having to pass in
+	// specific values
+	Usemd bool `long:"usemd" optional:"true"`
```

Comment:
Initialisms should be consistent case.  The way you have it technically is consistent case, but its not camel case.  Should be `UseMD`.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms